verifier/src/lib.rs

```diff
@@ -229,6 +229,26 @@ impl PkiPathSignatureVerifier {
     }
 }
 
+pub fn verify_signature(
+    alias: &Certificate,
```

Comment:
`alias` here is misleading since this could be any certificate. Probably got refactored from code that was checking the signature on an attestation?